apps/web/app/[lang]/sites/components/sitePreviewCard/SitePreviewCardMobile.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+  P2,
+  OfficeBuildingIcon,
+  L1,
+  Button,
+  ShareOutlineIcon,
+  CloseIcon,
+  P1,
+} from 'ui/core';
+import { ImageWithFallback } from '@components/utils/ImageWithFallback';
+import { SitePreviewProps } from './SitePreviewCard.types';
+import Link from 'next/link';
+import { LocaleProps, defaultLocale, useTranslations } from 'intl';
+
+export const SitePreviewCardMobile: React.FC<
+  SitePreviewProps & LocaleProps
+> = ({ site, onClose = () => {}, locale = defaultLocale }) => {
+  const t = useTranslations('sites.site');
+
+  return (
+    <>
+      <div className="flex flex-row w-full justify-between">
+        <ImageWithFallback
+          src={site.logo_url}
+          width={64}
+          height={64}
+          alt={'logo'}
+        />
+        <span onClick={onClose}>
+          <CloseIcon height={19} width={19} />
+        </span>
+      </div>
+
+      <P1
+        className="block font-bold text-base !my-3 !p-0"
+        label={site.name}
+      ></P1>
```

Comment:
```suggestion
      />
```
Use self closing tags when there is no children